src/notiDaemon/notiDaemon.vala

```diff
@@ -182,8 +182,15 @@ namespace SwayNotificationCenter {
                     noti_window.add_notification (param, this);
                 }
             }
-            // Only add notification to CC if it isn't IGNORED
-            if (state != NotificationStatusEnum.IGNORED) {
+            // Only add notification to CC if it isn't IGNORED and not transient
+            var transient_hint = hints.get("transient");
+            var transient = (transient_hint != null &&
+                transient_hint.is_of_type (GLib.VariantType.INT32) &&
+                    transient_hint.get_int32() == 1 ||
+                transient_hint.is_of_type (GLib.VariantType.BOOLEAN) &&
+                    transient_hint.get_boolean()
+            );
```

Comment:
This should be ran in the notiModel.vala file where all of the other hints are parsed

Reply:
So should I set a `transient` attribute and check its value here?

Reply:
> So should I set a `transient` attribute and check its value here?

Precisely! Just like all of the other hints :)

Reply:
Done,  thanks!